### ISSUE TITLE
NVDA does not warn about downgrading portable copies

### ISSUE THREAD

**Issue by SaschaCowley** [nvaccess/nvda#18291]
### Steps to reproduce:

1. Download 2 versions of NVDA (for instance, NVDA 2024.4.2 and 2025.1.1)
2. Create a portable copy with the newer version of NVDA.
3. Create a portable copy of the older version of NVDA in the same location as the portable copy created in step 2.

### Actual behavior:

NVDA notifies the user that there is already a copy of NVDA at that location, and asks them whether they would like to update it. The message reads:

> A portable copy already exists in the directory 'c:\nvda'. Do you want to update it?

### Expected behavior:

The user should be shown a similar message to that shown when attempting to downgrade an installed copy of NVDA. For example:

> You are attempting to create a portable copy of an earlier version than the version that already exists in the directory 'c:\nvda'. If you really wish to revert to an earlier version, you should choose a different directory.

### NVDA logs, crash dumps and other attachments:
N/A

#### NVDA installed/portable/running from source:
Portable

#### NVDA version:
Tested with NVDA 2024.4.2 and 2025.1.1.

#### Windows version:
Windows 11 Business Insider Preview Version 24H2 (OS Build 26200.5641) x64

#### Does the issue still occur after restarting your computer?
Yes

#### Have you tried any other versions of NVDA? If so, please report their behaviors.
No

#### If NVDA add-ons are disabled, is your problem still occurring?
Yes

#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
Yes
